package.json

```
@@ -3,6 +3,7 @@
   "version": "3.1.2",
   "description": "SDK for Auth0 API v2",
   "main": "src/index.js",
+  "module": "src/index.mjs",
```

Could you use conditional exports for this, I believe `module` is the "old" way of specifying esm exports for bundlers, and the standard node way is `exports`

```suggestion
  "exports": {
    "import": "./src/index.mjs",
    "require": "./src/index.js"
  },
```

(and also check that it resolves your issue - thanks!)

---

Yes, it makes sense to use exports...however it will require a major version update for this library. I went with the more conservative route of adding "module" but if you are willing to push a major version update, then we can switch it to exports.

---

Thanks @btakita - possibly a stupid question - why do you think it would be a breaking change?

---

I think it has to do with "preventing any other entry points besides those defined in ["exports"](https://nodejs.org/api/packages.html#exports)." in https://nodejs.org/api/packages.html#package-entry-points

I suppose that if *all* of the modules in the package were defined within exports, then it may not be a breaking change. I'm not sure though. I'll look more into an authoritative source on whether it would still be a breaking change if all modules in the package were defined in exports.

---

Ah ok, I didn't know that - let's stick with `"module"` then